sklearn/feature_extraction/text.py

```diff
@@ -488,7 +514,9 @@ class HashingVectorizer(BaseEstimator, VectorizerMixin, TransformerMixin):
         word boundaries; n-grams at the edges of words are padded with space.
 
         If a callable is passed it is used to extract the sequence of features
-        out of the raw, unprocessed input.
+        out of the raw, unprocessed input. If ``input`` is ``filename`` or
+        ``file``, the data is first read from the file and then passed to the
+        given callable analyzer.
 
```

Comment:
Should we note versionchanged? Is that appropriate for a bug fix?

Reply:
Not sure if versionadded would work here, but added the version in the docstring.